src/frontend/src/pages/PPQFormPage.tsx

```diff
@@ -0,0 +1,276 @@
+                <label>
+                  <input type="radio" value="no" />
+                  No
+                </label>
+              </div>
+              <div className="radio">
+                <label>
+                  <input type="radio" value="notsure" />
+                  I'm not sure
+                </label>
+              </div>
+            </div>
+            <div className="ppq-form-content">
+              <h3>4. Other factors</h3>
+              <span>
+                Does your initiative involve any of the following? Check all
+                that apply.
+              </span>
+              <div>
+                {OtherFactor.map((name, index) => {
+                  return (
+                    <div>
+                      <div>
+                        <input
+                          type="checkbox"
+                          id={`custom-checkbox-${index}`}
+                          name={name}
+                          value={name}
+                        />
+                        <label htmlFor={`custom-checkbox-${index}`}>
+                          {name}
+                        </label>
+                      </div>
+                    </div>
+                  );
+                })}
+              </div>
+            </div>
+
+            <div className="ppq-form-content">
+              {' '}
+              <h2>5. Start date </h2>
+              <div className="row">
+                <div className="form-group col-md-6">
+                  <span>
+                    Do you have a proposed go-live or start date for the
+                    initative?
+                  </span>
+                  <div>
+                    <label>
+                      <input type="radio" value="yes" checked={true} />
+                      Yes
+                    </label>
+                  </div>
+                  <div className="radio">
+                    <label>
+                      <input type="radio" value="no" />
+                      No
+                    </label>
+                  </div>
+                </div>
+
+                <div className="form-group col-md-6">
+                  <label>Proposed go-live or start date</label>
+                  <DatePicker
+                    placeholderText="yyyy-MM-dd"
+                    dateFormat="yyyy/MM/dd"
+                    //  selected={startDate}
+                     onChange={(date) => setStartDate(startDate)}
+                  />
+                </div>
+              </div>
+            </div>
+            <div className="horizontal-divider"></div>
+            <div className="form-buttons">
+              <button
+                className="btn-secondary btn-back"
+                onClick={handleBackClick}
+              >
+                Back
+              </button>
+              <button className="btn-primary btn-next">Submit</button>
+            </div>
+          </form>
+        </div>
+      </section>
+      <Footer />
```

Comment:
@binggaofw - we should not need to inject Headers and Footers in every page. 

Approving for now

Reply:
will do it in the integration ticket